python/src/scipp/compat/mantid.py

```diff
@@ -74,7 +74,7 @@ def make_variables_from_run_logs(ws):
         unit = lookup_units.get(units_string, sc.units.one)
         values = deepcopy(ws.run()[property_name].value)
 
-        if units_string and unit is None:
+        if units_string and unit == sc.units.one:
```

Comment:
Hmm, but there might be logs that ARE actually dimensionless?

Reply:
Okay, I think in that case the units_string is empty; don't think I've seen "dimensionless" or "one" as the units string. But I'll change it to be safe.

Reply:
Pushed unit test and fix for that situation